packages/cfd/src/Components/props.types.ts

```diff
@@ -66,6 +66,8 @@ export type TTradingPlatformAvailableAccount = {
     };
     shortcode: 'bvi' | 'labuan' | 'svg' | 'vanuatu' | 'maltainvest';
     sub_account_type: string;
+    account_type?: 'real' | 'demo';
+    landing_company_short?: 'bvi' | 'labuan' | 'svg' | 'vanuatu';
```

Comment:
I do not think if we're going to have all of these landing companies for deriv ez. We are just using the svg for now.